x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/utils/index.ts

```diff
@@ -5,30 +5,7 @@
  * 2.0.
  */
 
-import { VectorScoreBase, Vector } from '../types';
-
-export const getVectorScoreList = (vectorBaseScore: VectorScoreBase) => {
-  const result: Vector[] = [];
-  const v2Vector = vectorBaseScore?.V2Vector;
-  const v2Score = vectorBaseScore?.V2Score;
-  const v3Vector = vectorBaseScore?.V3Vector;
-  const v3Score = vectorBaseScore?.V3Score;
-
-  if (v2Vector) {
-    result.push({
-      version: '2.0',
-      vector: v2Vector,
-      score: v2Score,
-    });
-  }
-
-  if (v3Vector) {
-    result.push({
-      version: '3.0',
-      vector: v3Vector,
-      score: v3Score,
-    });
-  }
-
-  return result;
-};
+export * from './custom_sort_script';
```

Comment:
I don't have a preference about that so I reverted, I only think we need to have utils/helpers functions in their own files whenever possible, so I kept the `get_vector_score_list` and removed the `index` as it could be misleading.